wire-grpc-server-generator/src/test/golden/unitService.kt

```diff
@@ -0,0 +1,146 @@
+import com.google.protobuf.DescriptorProtos
```

Comment:
copyright

Reply:
This is a directory of "golden" generated files to compare test runs against, so can't really add a copyright here without adding extra logic to strip it out in the tests. However, I assume it is OK, as this is generated code, and non of the other files here have the copyright either?

Reply:
I see.
Maybe in the future, we could add the headers we add elsewhere to make it clear.

An example:
```
// Code generated by Wire protocol buffer compiler, do not edit.
// Source: squareup.wire.benchmarks.proto2.AllTypes in squareup/wire/benchmarks/all_types_proto2.proto
```

Reply:
Yeah, that would be good. I might see if I can open a second PR for that